site/specs/webrtc.yml

```diff
@@ -208,6 +221,85 @@ paths:
             rescue APIException => e
                 puts e.response_code
             end
+  /accounts/{accountId}/participants/{participantId}/dialogs:
+    post:
+      tags:
+        - Dialogs
+      summary: Create a Dialog
+      description: >-
+        Create a new Dialog for this participant.   The dialog will be used to
+        control the user-level state of the participant,  handling and managing
```

Comment:
What does "the user-level state of the participant" mean? User level as opposed to what other level(s)?